src/app/components/LiveLabel/index.styles.ts

```diff
@@ -1,13 +1,77 @@
-import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { css, Theme, keyframes } from '@emotion/react';
+
+const pulseAnimation = keyframes({
+  '0%': {
+    opacity: 1,
+  },
+  '50%': {
+    opacity: 0,
+  },
+  '100%': {
+    opacity: 1,
+  },
+});
 
 const styles = {
-  liveLabel: ({ palette, spacings, fontVariants }: Theme) =>
+  liveLabelText: ({ palette, spacings, fontVariants }: Theme) =>
     css({
       color: palette.LIVE_DARK,
       display: 'inline-block',
       marginInlineEnd: `${spacings.FULL}rem`,
       ...fontVariants.sansBold,
     }),
+  liveLabelCircle: ({ palette, spacings }: Theme) =>
+    css({
+      color: palette.LIVE_DARK,
+      borderRadius: '50%',
+      display: 'inline-block',
+      width: `${spacings.DOUBLE}rem`,
+      height: `${spacings.DOUBLE}rem`,
+      background: `radial-gradient(circle, transparent 53%, ${palette.LIVE_DARK} 53.5%, ${palette.LIVE_DARK} 54%)`,
+      marginInlineEnd: `${spacings.HALF}rem`,
+      position: 'relative',
+      transform: 'translate(0, 12%)',
+    }),
+
+  firstPromo: ({ mq, spacings }: Theme) =>
+    css({
+      [mq.GROUP_1_ONLY]: {
+        width: `${pixelsToRem(18)}rem`,
+        height: `${pixelsToRem(18)}rem`,
+      },
+      [mq.GROUP_2_ONLY]: {
+        width: `${spacings.TRIPLE}rem`,
+        height: `${spacings.TRIPLE}rem`,
+      },
+      [mq.GROUP_3_MIN_WIDTH]: {
+        width: `${pixelsToRem(28)}rem`,
+        height: `${pixelsToRem(28)}rem`,
+      },
+    }),
+
+  livelabelPulse: ({ palette }: Theme) =>
+    css({
+      '&:before': {
+        content: `''`,
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        borderRadius: '50%',
+        background: `radial-gradient(circle, ${palette.LIVE_DARK} 37%, ${palette.LIVE_DARK} 37.5%,transparent 38%)`,
+        '@media (prefers-reduced-motion: no-preference)': {
```

Comment:
I've tried switching off animations on my Samsung Flip 3 (on Chrome), and it is still pulsing for me. Keen to see if others have the same issue 😟

Reply:
cc @greenc05 - iPhone seems to work OK. I have also checked Public Service (on https://www.bbc.co.uk), which uses `@media screen and (prefers-reduced-motion: no-preference)`, and it continues to pulse for me. So, I am not really sure what we can do here, because the css query is correct.

Reply:
Works ok for me on Google Pixel with Chrome on the bbc home page

Reply:
OK great, that's a relief. Let me also restart my browser to see if that works 🤞

Reply:
And it worked! 😌 We'll need to make sure we do that during the swarm.